packages/twenty-website/src/app/_components/user-guide/UserGuideEditContent.tsx

```diff
@@ -0,0 +1,109 @@
+'use client';
+import styled from '@emotion/styled';
+
+import { StyledButton } from '@/app/_components/ui/layout/header/styled';
+import { Theme } from '@/app/_components/ui/theme/theme';
+
+const StyledContainer = styled.div`
+  margin-top: 64px;
+  padding: 32px;
+  border: 2px solid #141414;
+  border-radius: 16px;
+  display: flex;
+  flex-direction: row;
+
+  h1 {
+    margin: 0px;
+    font-size: ${Theme.font.size.lg};
+  }
+
+  p {
+    font-size: ${Theme.font.size.xs};
+    margin: 32px 0px;
+    color: ${Theme.text.color.tertiary};
+  }
+
+  @media (min-width: 900px) {
+    padding: 32px 0px 0px 32px;
+  }
+`;
+
+const StyledButtonText = styled.div`
+  margin-left: 8px;
+  text-decoration: none;
+`;
+
+const StyledRightColumn = styled.div`
+  display: flex;
+  border-bottom-right-radius: 16px;
+  overflow: hidden;
+  min-width: 40%;
+  @media (max-width: 900px) {
+    display: none;
+  }
+
+  @media (min-width: 1530px) {
+    justify-content: end;
+  }
+
+  img {
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 100%;
+  }
+`;
+
+const StyledButtonContainer = styled.div`
+  text-decoration: none;
+
+  @media (min-width: 900px) {
+    padding-bottom: 32px;
+  }
+`;
+
+export default function UserGuideEditContent() {
+  return (
+    <StyledContainer>
+      <div>
+        <h1>Noticed something to modify ?</h1>
+        <p>
+          As an open-source company, we invite modifications to our GitHub user
+          guide for real-time updates, accuracy improvement, and broader
+          comprehensibility through community contributions.
```

Comment:
That text probably isn't from you but could be improved. Since you're english-native would you mind making it feel more natural? Feel free to always take the liberty to edit. The more straightforward, the better, let's not overplay (e.g. "real time update", "comprehensibility", etc)